lib/exercism/participation_stats.rb

```diff
@@ -0,0 +1,99 @@
+require 'date'
+
+class ParticipationStats
+  PRE_GAMIFICATION_COMPARISON_DATE = '2017-03-20'
+  GAMIFICATION_START_DATE          = '2017-03-27'
+  GAMIFICATION_WITHDRAWAL_DATE     = '2017-04-10'
+  GAMIFICATION_EXPERIMENT_END_DATE = '2017-04-24'
+
+  attr_reader :date_range, :gamification_markers, :experiment_group
+
+  def self.experiment_complete?
+    Time.now > Date.parse(GAMIFICATION_EXPERIMENT_END_DATE) + 1.day
+  end
+
+  def initialize(date_range, gamification_markers: false, experiment_group: nil)
+    @date_range = date_range
+    @gamification_markers = gamification_markers
+    @experiment_group = experiment_group
+  end
+
+  def results
+    result = {
```

Comment:
Is the order important here? `date[0]` refer to `daily_review_count[0]` which refer to `daily_review_count[0]` and so on? If so, we could create a struct like: `DailyReview = Struct.new(:date, :count, :length)` and instantiate it when iterating through the results.